examples/connectivity/compare_resting_state_decomposition.py

```diff
@@ -0,0 +1,81 @@
+"""
+Comparison of Dictionary Learning and ICA for doing group analysis of
+resting-state fMRI
+=====================================================
+
+An example applying dictionary learning and ICA to resting-state data,
+comparing resulting components using 4D plotting.
+
+Dictionary learning is a sparsity based decomposition method for extracting
+spatial maps. It extracts maps that are naturally sparse and usually cleaner
+than ICA
+
+    * Gael Varoquaux et al.
+    Multi-subject dictionary learning to segment an atlas of brain spontaneous
+    activity
+    Information Processing in Medical Imaging, 2011, pp. 562-573, Lecture Notes
+    in Computer Science
+
+https://hal.inria.fr/inria-00588898/en/
+"""
+
+### Load ADHD rest dataset ####################################################
```

Comment:
Could you rebase on master, and then use the new style of division (for notebook-style sphinx gallery):
<pre>
#######################################################################
# Load ADHD rest dataset
</pre>
throughout the example.